### PR TITLE
bug: update pathforajs to use jstag3 callbacks

### PR DESCRIPTION
**Pull request by RaeSmith1331**

### CODE REVIEW

File: dist/pathfora.js [hunk 1]
```diff
@@ -1014,11 +1014,23 @@
...
     } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
+
+      var version = parseInt(window.jstag.version, 10);
       // > jstag 3.0.0
-      var entity = window.jstag.getEntity();
-      if (entity.data && entity.data.user) {
-        cb(entity.data.user);
-        return;
+      if (version >= 3) {
+        window.jstag.send('entityReady', function () {
```
Comment (ashyablok) [inline dist/pathfora.js:1021]: I have never seen this format before. 🤔 could we not do:

`window.jstag.entityReady(entity => ...)`

Comment (RaeEversmann) [reply dist/pathfora.js]: updated

File: dist/pathfora.js [hunk 2]
```diff
@@ -1014,11 +1014,29 @@
       cb(window.lio.data);
       return;
     } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
-      // > jstag 3.0.0
-      var entity = window.jstag.getEntity();
-      if (entity.data && entity.data.user) {
-        cb(entity.data.user);
-        return;
+      if (window.jstag.version) {
```
Comment (RaeEversmann) [inline dist/pathfora.js:1017]: I don't think jstag2 has the version field, so we need a check here.

**Review by ashyablok — APPROVED** [review]

**Review by humanchimp — APPROVED** [review]

**Pull request merged**